lib/core/class-groups-user-group.php

```diff
@@ -152,7 +152,7 @@ public static function read( $user_id, $group_id ) {
 		if ( $user_group !== null ) {
 			$result = $user_group;
 		}
-		return $result;
+		return apply_filters( 'groups_user_read', $result, $user_id, $group_id );
```

Comment:
We need to review this filter `groups_user_read` because it does not adhere to our naming conventions, noted for review and inclusion after adjustment.